test/e2e/go.sum

```diff
@@ -542,13 +543,18 @@ github.com/onsi/ginkgo v1.12.1/go.mod h1:zj2OWP4+oCPe1qIXoGWkgMRwljMUYCdkwsT2108
 github.com/onsi/ginkgo v1.14.0/go.mod h1:iSB4RoI2tjJc9BBv4NKIKWKya62Rps+oPG/Lv9klQyY=
```

Comment:
we cloud clean older version, right?

Reply:
We can't. 
I try to remove it from the `go.sum`, but after re-running `go mod tidy`, it will come back again.